Source/Urho3D/Physics/RaycastVehicle.cpp

```diff
@@ -34,6 +34,8 @@
 namespace Urho3D
 {
 
+const IntVector3 RaycastVehicle::DEFAULT_COORDINATE_SYSTEM(0, 1, 2);
```

Comment:
Since 3P3 = 6, do you think it makes sense to declare all the possible permutations here?